internal/backend/dispatcher/eventsworkflow.go

```diff
@@ -210,12 +210,14 @@ func (d *dispatcher) signalWorkflows(ctx context.Context, event sdktypes.Event)
 	return nil
 }
 
-func (d *dispatcher) eventsWorkflow(ctx workflow.Context, input eventsWorkflowInput) (*eventsWorkflowOutput, error) {
-	logger := workflow.GetLogger(ctx)
-	logger.Info("started events workflow", "event_id", input.EventID)
+func (d *dispatcher) eventsWorkflow(wctx workflow.Context, input eventsWorkflowInput) (*eventsWorkflowOutput, error) {
+	logger := workflow.GetLogger(wctx)
 
+	logger.Info("started events workflow", "event_id", input.EventID)
 	z := d.Z.With(zap.String("event_id", input.EventID.String()))
-	event, err := d.Services.Events.Get(context.TODO(), input.EventID)
+
+	ctx := context.Background()
```

Comment:
I think there is a function to get go context from workflow context, maybe it's better then starting a new context ?
could you check please?

Reply:
maybe. But in old code new context was started many times when it needed to be passed. I'm just going 1 step a time - I create it once and pass it through functions. Maybe we could do it event better and obtain it from the workflow context. At least this change was checked by me when using tagging components, and using context from workflow - isn't. So I prefer to stay meanwhile with that